src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -484,6 +484,40 @@ async Task<DurableOrchestrationStatus> IDurableOrchestrationClient.GetStatusAsyn
             return await GetDurableOrchestrationStatusAsync(state, this.client, showHistory, showHistoryOutput);
         }
 
+        /// <inheritdoc />
+        async Task<List<DurableOrchestrationStatus>> IDurableOrchestrationClient.GetStatusAsync(IEnumerable<string> instanceID, bool showHistory, bool showHistoryOutput, bool showInput)
```

Comment:
Since the input here is an array, I think we should rename `IEnumerable<string> instanceID` to `IEnumerable<string> instanceIDs` instead, to denote its a collection of values :)